base_layer/core/src/consensus/network.rs

```diff
@@ -35,7 +35,7 @@ pub enum Network {
 }
 
 impl Network {
-    pub fn create_consensus_constants(self) -> ConsensusConstants {
+    pub fn create_consensus_constants(&self) -> ConsensusConstants {
```

Comment:
Clippy would actually complain about this because in this case it's actually cheaper to pass in a copy of the value than a new reference. https://rust-lang.github.io/rust-clippy/master/#trivially_copy_pass_by_ref - super pedantic though.